src/Components/Modal/Modal.scss

```
@@ -0,0 +1,100 @@
+.modal {
+  display: none;
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  z-index: 99;
+  background-color: rgba(0, 0, 0, 0.6);
+
+  button {
+    outline: none;
+    cursor: pointer;
+    border: 0;
+  }
+  & > section {
+    width: 90%;
+    max-width: 450px;
+    margin: 0 auto;
+    border-radius: 0.3rem;
+    background-color: #fff;
+    animation: modal-show 0.3s;
+    overflow: hidden;
+
+    & > header {
+      all: initial;
+      position: relative;
+      padding: 16px 64px 16px 16px;
+      font-weight: 700;
+      margin: auto !important;
+
+      div.close {
```

```suggestion
      .close {
```
클래스 셀렉터만으로도 충분히 요소가 특정될 것 같습니다

---

클래스 셀렉터만 남겨두고 태그 셀렉터는 삭제하였습니다 😀